openedx/api.py

```diff
@@ -921,3 +955,19 @@ def validate_username_with_edx(username):
         raise EdxApiRegistrationValidationException(username, resp)
     result = resp.json()
     return result["validation_decisions"]["username"]
+
+
+def bulk_retire_edx_users(usernames):
+    """
+    Bulk retires edX users.
+
+    This method calls edX bulk enrollment API to initiate the user retirement on edX.
```

Comment:
```suggestion
    This method calls edX bulk retirement API to initiate the user retirement on edX.
```

Reply:
Can we also mention the possible states for retirement in edX? Or would that be confusing information?

Reply:
> Can we also mention the possible states for retirement in edX? Or would that be confusing information?

Retirement states are dynamic and configured by a setting in edX `RETIREMENT_STATES`. Pending is the initial state that we wanted to create for a user and its mentioned on the next line.